DQM/L1TMonitorClient/python/L1TEMTFEventInfoClient_cfi.py

```diff
@@ -0,0 +1,233 @@
+# L1 Trigger Event Info client cfi
```

Comment:
This file can be removed since it is replaced by L1TStage2EMTFEventInfoClient_cfi.py